
From: Arjan van de Ven <arjanv@redhat.com>

LOOP_CHANGE_FD is an ULONG compatible ioctl, basically same calling
convention as LOOP_SET_FD; mark it as such in the compat ioctl list.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/compat_ioctl.h |    1 +
 1 files changed, 1 insertion(+)

diff -puN include/linux/compat_ioctl.h~mark-loop_change_fd-as-an-ulong-compat-ioctl include/linux/compat_ioctl.h
--- 25/include/linux/compat_ioctl.h~mark-loop_change_fd-as-an-ulong-compat-ioctl	Wed Aug  4 15:55:37 2004
+++ 25-akpm/include/linux/compat_ioctl.h	Wed Aug  4 15:55:37 2004
@@ -386,6 +386,7 @@ COMPATIBLE_IOCTL(DVD_AUTH)
 COMPATIBLE_IOCTL(PACKET_CTRL_CMD)
 /* Big L */
 ULONG_IOCTL(LOOP_SET_FD)
+ULONG_IOCTL(LOOP_CHANGE_FD)
 COMPATIBLE_IOCTL(LOOP_CLR_FD)
 COMPATIBLE_IOCTL(LOOP_GET_STATUS64)
 COMPATIBLE_IOCTL(LOOP_SET_STATUS64)
_
